WordPress/src/main/java/org/wordpress/android/ui/plans/PlansUtils.java

```diff
@@ -94,6 +94,24 @@ public static String getIconUrlForPlan(Plan plan, int iconSize) {
         return PhotonUtils.getPhotonImageUrl(plan.getIconUrl(), iconSize, iconSize);
     }
 
+    public static boolean isFreePlan(Plan plan) {
+        return plan.getProductSlug().equals("free_plan") ||
+               plan.getProductSlug().equals("jetpack_free");
+    }
+
+    /**
+     * Weather the plan ID is a free plan.
+     *
+     * 2002 = Jetpack free
+     * 1    = WordPress.com free
+     *
+     * @param planID - The plan ID
+     * @return boolean - true if the current blog is on a free plan.
+     */
+    public static boolean isFreePlan(long planID) {
+        return planID == 2002 || planID == 1;
```

Comment:
Let's use the values defined in `PlanConstants.java` instead of `2002` and `1` here.